scipy/special/add_newdocs.py

```diff
@@ -25,6 +25,55 @@ def add_newdoc(place, name, doc):
     Private function; do not use.
     """)
 
+add_newdoc("scipy.special", "_cosine_cdf",
```

Comment:
Feels strange to be arguing against documentation but... since this is an internal function maybe it should just link to the stats methods it's going to be used for?

Reply:
The functions will be used within the distribution `scipy.stats.cosine`, and these (for better or worse) already have standard docstrings for the `cdf` and `ppf` methods.  Is there something *wrong* with providing docstrings here? (I don't think there is!)